types/ApiRequests.ts

```diff
@@ -127,6 +127,19 @@ export interface SubmitAttendanceForUsersRequest {
   asStaff?: boolean;
 }
 
+export interface UserAccessPatches {
+  accessType?: string;
+}
+
+export interface UserAccessUpdates {
+  user: string;
+  newAccess: UserAccessPatches;
```

Comment:
This was done because `UserAccessPatches` is supposed to be a Partial User Model. That way I can use the `upsertUser()` function to easily make changes to the user (if you look at `UserAccessPatches`, it's basically just the `accessType` as an optional param.